test/KoreBuild.FunctionalTests/Utilities/TestApp.cs

```diff
@@ -41,22 +51,22 @@ void Write(object sender, DataReceivedEventArgs e)
             {
                 cmd = "cmd.exe";
                 arguments.Add("/C");
-                arguments.Add(@".\build.cmd");
+                arguments.Add($@".\{script}.cmd");
             }
             else
             {
                 cmd = "bash";
-                arguments.Add("./build.sh");
+                arguments.Add($"./{script}.sh");
             }
 
             arguments.AddRange(new[]
             {
-                "-ToolsSource", _toolsSource,
-                "-Update"
+                "-s", _toolsSource,
```

Comment:
Nit: I personally prefer the readability of the long option name. The short option is good for command-line shorthand, but harder to interpret from logs and code.